lib/p2p/errors.ts

```diff
@@ -41,6 +42,10 @@ const errors = {
     message: `node (${nodePubKey}) is not connected`,
     code: errorCodes.NOT_CONNECTED,
   }),
+  NODE_TOR_ADDRESS: (nodePubKey: string, address: Address) => ({
+    message: `node (${nodePubKey}) at ${addressUtils.toString(address)} is tor node`,
```

Comment:
How about a message like "can't connect to node {node} at tor address {address} because tor is disabled"